### DISCUSSION TITLE
Status "In progress" as I completed learning modules before this internship

### DISCUSSION THREAD

**Discussion by NithinShanmugamV** [github/india#357] [Students]
Hi there. I completed a couple of cloud learning modules a few months ago to build cloud knowledge. Now, the FRT portal is asking me to do it from the start and I am completely fine with doing all the modules again. However, the problem is that some modules are marked as finished. So when I re-start, the portal cannot track my learning. I mailed my issue to support but I didn't get a response.  Does anyone have an idea of what to do next?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by NithinShanmugamV** [reply]
I have asked in the above discord group too. I did not receive an answer.

**Reply by TheSuranaverse** [reply]
@NithinShanmugamV pls ping me there
my username is Neklaustares
